From 724bbcb82be960168f6636969e8ea4e7fdb99bb8 Mon Sep 17 00:00:00 2001
From: Luwei Zhou <b45643@freescale.com>
Date: Thu, 18 Sep 2014 13:15:43 +0800
Subject: [PATCH 0364/5242] MLK-11474 ARM: dts: i.mx6sl-evk: Add mma8450
 sensor dts support on i.MX6SL-EVK platform.

commit  4b054e93f98c9bc8bf7007864580fe034b5b60d4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Support MMA8450 sensor on i.MX6SL-EVK

Signed-off-by: Luwei Zhou <b45643@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index 33ce59b..5fe75be 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -249,6 +249,11 @@
 			};
 		};
 	};
+
+	mma8450@1c {
+		compatible = "fsl,mma8450";
+		reg = <0x1c>;
+	};
 };
 
 &i2c2 {
-- 
1.7.9.5

